packages/cli-dashboard/src/components/siteReport/tabs/cookies/cookiesListing/index.tsx

```diff
@@ -86,6 +86,7 @@ const CookiesListing = ({
         className="h-full flex"
       >
         <CookieTable
+          //@ts-ignore
```

Comment:
Please add the reason for using '//@ts-ignore'.

Reply:
Resolved in latest commits.